docs/releases/1.24-NOTES.md

```diff
@@ -12,6 +12,8 @@ This is a document to gather the release notes prior to the release.
 
 * Support for Kubernetes version 1.18 has been removed.
 
+* Support for Aliyun/Alibaba Cloud has been removed.
```

Comment:
There are some more reference in various docs to AliCloud/Aliyun/Alibaba. Should we remove those too?

Reply:
I didn't find any for "Aliyun" or "Alibaba" outside of release notes. I just removed references to "AliCloud".

Reply:
https://github.com/kubernetes/kops/blob/master/ROADMAP.md#upcoming-features ?

Reply:
Can be done later. That whole file need work.